packages/core/src/schema/models/evaluationConfigurationNumerical.ts

```diff
@@ -7,8 +7,8 @@ export const evaluationConfigurationNumerical = latitudeSchema.table(
   'evaluation_configuration_numerical',
   {
     id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
-    minValue: numeric('min_value').notNull(),
-    maxValue: numeric('max_value').notNull(),
+    minValue: bigint('min_value', { mode: 'number' }).notNull(),
+    maxValue: bigint('max_value', { mode: 'number' }).notNull(),
```

Comment:
this type change does not cause data removal no?

Reply:
No, there's still no rows whatsoever in this table. There will be from now on tho